sdk/lib/collection/hash_map.dart

```diff
@@ -52,7 +52,7 @@ typedef _Hasher<K> = int Function(K object);
 /// planets.addAll({4: 'Mars'});
 /// final gasGiants = {6: 'Jupiter', 5: 'Saturn'};
 /// planets.addEntries(gasGiants.entries);
-/// print(planets); // fx {5: Saturn, 6: Jupiter, 3: Earth, 4: Mars}
+/// print(planets); // fx {3: Earth, 4: Mars, 6: Jupiter, 5: Saturn}
```

Comment:
That's not a bug. `HashMap` is unordered which means any output is possible here. That's why it says `fx`. 

Consider instead

```
/// print(planets);  // fx {5: Saturn, 6: Jupiter, 3: Earth, 4: Mars}, no particular order of keys is guaranteed
```

or something similar

cc @lrhn

Reply:
Done